doc/reference/configuration/configuration_reference.rst

```diff
@@ -1249,6 +1217,149 @@ The ``database`` section defines database-specific configuration parameters, suc
     | Default: false
     | Environment variable: TT_DATABASE_USE_MVCC_ENGINE
 
+..  _configuration_reference_fiber:
+
+fiber
+-----
+
+The ``fiber`` section describes options related to configuring :ref:`fibers, yields, and cooperative multitasking <concepts-coop_multitasking>`.
+
+..  NOTE::
+
+    ``fiber`` can be defined in any :ref:`scope <configuration_scopes>`.
+
+* :ref:`fiber.io_collect_interval <configuration_reference_fiber_io_collect_interval>`
+* :ref:`fiber.too_long_threshold <configuration_reference_fiber_too_long_threshold>`
+* :ref:`fiber.worker_pool_threads <configuration_reference_fiber_worker_pool_threads>`
+* :ref:`fiber.slice.* <configuration_reference_fiber_slice_options>`
+* :ref:`fiber.top.* <configuration_reference_fiber_top_options>`
+
+..  _configuration_reference_fiber_io_collect_interval:
+
+..  confval:: fiber.io_collect_interval
+
+    The :ref:`fiber <app-fibers>` will sleep for the defined number of seconds
+    between iterations of the event loop. Can be used to reduce CPU load in deployments
+    where the number of client connections is large, but requests are not so frequent
+    (for example, each connection issues just a handful of requests per second).
+
+    |
+    | Type: number
+    | Default: box.NULL
+    | Environment variable: TT_FIBER_IO_COLLECT_INTERVAL
+
+..  _configuration_reference_fiber_too_long_threshold:
+
+..  confval:: fiber.too_long_threshold
+
+    If processing a request takes longer than the given value (in seconds),
+    the fiber warns about it in the log.
+    Has effect only if :ref:`log.level <configuration_reference_log_level>`
+    is greater than or equal to 4 (``warn``).
+
+    |
+    | Type: number
+    | Default: 0.5
+    | Environment variable: TT_FIBER_TOO_LONG_THRESHOLD
+
+..  _configuration_reference_fiber_worker_pool_threads:
+
+..  confval:: fiber.worker_pool_threads
+
+    The maximum number of threads to use during execution
+    of certain internal processes (currently
+    :ref:`socket.getaddrinfo() <socket-getaddrinfo>` and
+    :ref:`coio_call() <c_api-coio-coio_call>`).
+
+    |
+    | Type: number
+    | Default: 4
+    | Environment variable: TT_FIBER_WORKER_POOL_THREADS
+
+.. _configuration_reference_fiber_slice_options:
+
+fiber.slice.*
+~~~~~~~~~~~~~
+
+This section describes options related to configuring
+:ref:`fiber slices <fibers_limit_execution_time>`.
+
+*   :ref:`fiber.slice <configuration_reference_fiber_slice>`
+
+    * :ref:`fiber.slice.warn <configuration_reference_fiber_slice_warn>`
+    * :ref:`fiber.slice.err <configuration_reference_fiber_slice_err>`
+
+..  _configuration_reference_fiber_slice:
+
+..  confval:: fiber.slice
```

Comment:
Let's drop this option completely. Having the `fiber.slice.*` section is enough to group slice options. Also, there is no `TT_FIBER_SLICE` env variable:

<img width="664" alt="image" src="https://github.com/tarantool/doc/assets/38073144/79b048ff-c1eb-47c5-b7e1-063ea6883889">

The same for `fiber.top`:

<img width="620" alt="image" src="https://github.com/tarantool/doc/assets/38073144/87d348bf-1f36-41fb-9c86-9e52599393b2">